### PR TITLE
Visual tweaks from feedback

### PR DESCRIPTION
**Pull request by c5inco**

### CODE REVIEW

**Comment by c5inco** [comment]
Ready for review and merge

**Comment by adebruine** [comment]
looks like on mobile the bottom of the key gets cut off. otherwise, looks good to me!

Screen Shot 2021-02-07 at 10 06 22 AM

File: src/styles/_header-title.scss [hunk 1]
```diff
@@ -18,11 +18,6 @@ header#aemp-titlebox {
     width: unset;
     right: 10px;
   }
-
```
Comment (adebruine) [inline src/styles/_header-title.scss:21]: we might want to keep a condition like this in (see my comment on PR)

Comment (c5inco) [reply src/styles/_header-title.scss:21]: Yes good catch! Made the tweak to always have space when `max-width: 900px`

See this [commit](https://github.com/antievictionmappingproject/covid-19-map-react/pull/27/commits/cc97b08a0b1459e40d972d2a4af7b5e5589cce60)

**Pull request merged**